services/motion/builtin/builtin_test.go

```diff
@@ -1219,38 +1261,10 @@ func TestMoveOnGlobe(t *testing.T) {
 			MovementSensorName: injectedMovementSensor.Name(),
 			Heading:            -10000000,
 			Destination:        dst,
-			Extra:              extra,
-		}
-		executionID, err := ms.MoveOnGlobe(ctx, req)
-		test.That(t, err, test.ShouldBeNil)
-		test.That(t, executionID, test.ShouldNotResemble, uuid.Nil)
-	})
-
-	t.Run("is able to reach a nearby geo point when the motion configuration is empty", func(t *testing.T) {
-		injectedMovementSensor, _, fakeBase, ms := createMoveOnGlobeEnvironment(ctx, t, gpsPoint, nil, 5)
-		defer ms.Close(ctx)
-		req := motion.MoveOnGlobeReq{
-			ComponentName:      fakeBase.Name(),
-			MovementSensorName: injectedMovementSensor.Name(),
-			Heading:            90,
-			Destination:        dst,
-			MotionCfg:          &motion.MotionConfiguration{},
```

Comment:
this test is removed since specifying a PlanDevM would mean we no longer have empty motion configuration.
I can adjust the destination so that this test remains we think that is a good idea.

Reply:
1. please add back the nil config case 
2. lets change the definition of `dst` to be slightly more than 2600m away from the robot position so that we both don't increase tests time & don't need to have different dst when the test case is not about changing dst.